app/views/devise/confirmations/new.html.erb

```diff
@@ -1,27 +1,67 @@
 <% title "Confirm your email" %>
 <% proper_email = params[:email] || (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
-<div class="crayons-layout crayons-layout--limited-l align-center mt-9 mb-10 p-3">
+<main id="main-content">
+  <% if FeatureFlag.enabled?(:creator_onboarding) %>
+    <%= javascript_packs_with_chunks_tag "confirmationEmail", defer: true %>
+    <div class="crayons-layout crayons-layout--limited-l" id="email-confirmation">
+      <div class="flex flex-col items-center justify-center">
+        <%= render "devise/shared/error_messages", resource: resource %>
 
-  <% if came_from_sign_up? %>
-    <h1 class="fs-5xl mb-5">Welcome to <%= community_name %>! 🎉</h1>
-  <% end %>
+        <%= inline_svg_tag("mail.svg", aria: true, title: "Email") %>
+        <div class="fs-3xl fw-bold">Great! Now confirm your email address.</div>
 
-  <% if proper_email.present? %>
-    <h3 class="mb-5">A confirmation email has been sent to <a href="mailto:<%= proper_email %>"><%= proper_email %></a></h3>
-  <% end %>
+        <% if proper_email.present? %>
+          <div class="fs-xl color-base-70">
+            <div>We've sent an email to <span class="fw-bold"><%= proper_email %></span></div>
+            <div>Click the button inside to confirm your email</div>
+          </div>
+        <% end %>
+      </div>
+      <div class="bg-base-10 p-4 self-end mx-auto mb-6">
+        <button class="color-accent-brand text-underline cursor-pointer js-confirmation-button border-none" role="button">Click here</button>if you didn't get the email...
+      </div>
+      <%= inline_svg_tag("forem-background.svg", aria: true, title: "forem background", class: "absolute bottom-0 right-0") %>
```

Comment:
Is there a way to make this SVG fall behind the other elements in the page? Maybe with a z-index value?

When using a mobile sized screen the SVG makes clicking/tapping the button impossible, so there would be no way to request another email if using a small screen.

<img width="1053" alt="Screen Shot 2021-07-29 at 16 03 32" src="https://user-images.githubusercontent.com/6045239/127571598-78e0339b-f3cb-4568-96cd-4608d1122cb8.png">

Reply:
Thanks for catching this @fdoxyz!

Unfortunately the z-index only works on positioned elements and the background image is absolute, so its to no avail :( 

I also tried moving the SVG element above all the other elements in the DOM, and using a `background-image` property instead. None of these worked so an alternate solution was to hide this background on smaller screens. I did that for now and will continue looking for a fix.